check-licenses/action.yml

```diff
@@ -161,6 +168,19 @@ runs:
         wget https://raw.githubusercontent.com/ansys/actions/main/check-licenses/accepted-licenses.txt
         wget https://raw.githubusercontent.com/ansys/actions/main/check-licenses/ignored-packages.txt
 
+    - name: "Process whitelisted packages provided on input"
+      shell: bash
+      run: |
+        if [[ -n "${{ inputs.whitelist-license-check }}" ]]; then
+          echo "Whitelisted packages: ${{ inputs.whitelist-license-check }}"
```

Comment:
The content of the ``ignored-packages.txt`` file is printed out: see https://github.com/ansys/pyansys-tools-report/actions/runs/11177206820/job/31072221914?pr=219#step:2:829 as an example

The reasons why we are ignoring some of them is already explained in the description I think but we can improve it if you want... what do you have in mind?

See https://github.com/ansys/actions/blob/0065b8b47ad1c1af41ec4279c254716854cf48a1/check-licenses/action.yml#L26-L66 and the rendering:

![image](https://github.com/user-attachments/assets/df28ec34-2668-49bd-87a8-8d2b82fed7a4)

Reply:
Just put the content of the description.
It is just for users that could find the file without knowing about the existence of the action.
Some contributors might see this file without knowing what is its intent because they don't know about all the actions we have.